app/views/docs/permissions.phtml

```diff
@@ -44,60 +43,62 @@
     </tr>
+    </tbody>
+</table>
+
+<h2 id="permission-types"><a href="/docs/permissions#permission-types">Permission Types</a></h2>
+<p>In Client and Server SDKs, you will find a <b>Permission</b> class with helper methods for each role described below.</p>
+<table class="full text-size-small">
+    <thead>
     <tr>
-        <td>team:[TEAM_ID]</td>
-        <td>Access to any member of the specific team. To gain access to this permission, the user must be the team creator (owner), or receive and accept an invitation to join this team.</td>
+        <td style="width: 180px">Type</td>
+        <td>Description</td>
     </tr>
+    </thead>
+    <tbody>
     <tr>
-        <td>team:[TEAM_ID]/[ROLE]</td>
-        <td>Access to any member who possesses a specific role in a team. To gain access to this permission, the user must be a member of the specific team and have the given role assigned to him or her. Team roles can be assigned when inviting a user to become a team member.</td>
+        <td>Permission.read()</td>
+        <td>Access to view a resource. Does not apply to functions.</td>
     </tr>
     <tr>
-        <td>member:[MEMBER_ID]</td>
-        <td>Access to a specific member of a team. Unlike the basic user permission, this permission will only be valid as long as the user is still an active member of the specific team. To view user member ID, fetch the team <a href="/docs/client/teams#teamsGetMemberships">members list</a>.</td>
+        <td>Permission.create()</td>
+        <td>Access to create new resources. Only applies to collections and buckets, applying this type of access to other resources results in an error. Does not apply to functions.</td>
     </tr>
     <tr>
-        <td>
-            role:guest
-            <p class="margin-top-small text-fade">version >= 0.7</p>
-        </td>
-        <td>Access to any guest user. Logged in users don't have access to this role.</td>
+        <td>Permission.update()</td>
+        <td>Access to change a resource, but not remove or create new resources. Does not apply to functions.</td>
     </tr>
     <tr>
-        <td>
-            role:member
-            <p class="margin-top-small text-fade">version >= 0.7</p>
-        </td>
-        <td>Access to any logged in user. A logged in user is a user with a valid session. Logged in users don't have access to `role:guest` role. For wildcard access use `*` role.</td>
+        <td>Permission.delete()</td>
+        <td>Access to remove a resource. Does not apply to functions.</td>
     </tr>
     <tr>
-        <td>
-            *
-            <p class="margin-top-small text-fade text-danger">version <= 0.11 (deprecated)</p>
-        </td>
-        <td>Wildcard permission. Gives anyone read or write access.</td>
+        <td>Permission.execute()</td>
```

Comment:
We don't have an execute method / permission as part of the SDK. 

The execute permission on the create function endpoint directly takes an array of roles 
https://github.com/appwrite/appwrite/pull/3700/files#diff-3c8abdcca7e143e051f729255409460b7fffbb41a3cf8dcc11c57651243662ecR60

Reply:
Removed